geoips/commandline/geoips_get.py

```diff
@@ -0,0 +1,342 @@
+# # # This source code is protected under the license referenced at
+# # # https://github.com/NRLMMD-GEOIPS.
+
+"""GeoIPS CLI "get" command.
+
+Retrieves the appropriate family/interface/package/plugin based on the args provided.
+"""
+
+from importlib import metadata, resources, import_module
+
+import yaml
+
+from geoips.commandline.geoips_command import GeoipsCommand, GeoipsExecutableCommand
+from geoips.create_plugin_registries import format_docstring
+from geoips import interfaces
+
+
+class GeoipsGetFamily(GeoipsExecutableCommand):
+    """Get Command which retrieves and outputs information about a GeoIPS Family.
+
+    This is called via `geoips get family <interface_name> <family_name>`. Data included
+    when calling this command is shown below, outputted in a yaml-based format.
+    """
+
+    name = "family"
+    command_classes = []
+
```

Comment:
Should this be ddeleted?  Is this the one that was replaced with geoips_describe?